docs/loadbalancer/index.rst

```diff
@@ -0,0 +1,88 @@
+Load Balancer Tutorial
+=====================================
+
+Getting Started
+-----------------
+- This tutorial assumes that you have access to `CloudLab <https://cloudlab.us/>`_ and a basic working knowledge of CloudLab and `SSH <https://www.ssh.com/ssh/>`_
+
+Overview
+-----------------
+- In the following tutorial, we will explore a means of deploying and testing ONVM’s example Layer-3 round-robin load balancer. To do this, we will instantiate a Cloudlab experiment using the Simple_Load_balancer profile; this topology (shown below) includes two backend servers and a single client, in addition to the ONVM load balancer.
+
+  .. image:: ../images/lb-1.png
+
+- After completion of the step-by-step guide below, you may use a number of packet generation tools to test the successful distribution of packets across multiple servers. The tools covered in this tutorial include iPerf and Pktgen, the latter being further described in the previous tutorial. 
+
+Cloudlab Node Setup
+-----------------
+- Open Cloudlab and start a new experiment. When prompted to choose a profile, select :code:`Simple_Load_Balancer`.
+- Begin by SSHing into each node within the experiment, and download the **Simple Load Balancer Topology Template** `here <https://github.com/jettjacobs/openNetVM/blob/docs/docs/loadbalancer/ONVM_LB_TopologyDoc.pdf>`_. If you are using any Apple product to complete this tutorial, avoid using Preview as your PDF editor; autofill scripts will not apply. Google Chrome or Adobe Acrobat are viable alternatives.
+- For every node, use :code:`ifconfig` to view all available network interfaces. Record the appropriate name, IPv4 (inet), and MAC address (ether) for each network interface in the Topology Template, as shown below. The ONVM_LB node requires the use of two ports: one for connection to the client and one for connecting to the servers. It is recommended that you use the 10-Gigabit SFI/SFP+ network connections. Port IDs will be handled later.
+
+  .. image:: ../images/lb-2.png
+
```

Comment:
@andreaseno I have added a shortened version of the binding instructions to the tutorial, as shown below. These were all of the steps that I felt were important to list, though I can add more detail (or alternatively I can link the three node tutorial) if you feel that the instructions are not specific enough. In regards to the new `go.sh` command for the ARP NF, where would I find this? The documentation for the ARP NF under the develop and master branch still seems to resemble the topology template, and I couldn't find any pull request which presents a different configuration of command-line arguments. Perhaps I am looking in the wrong place. 

1) Unbind the connected NICs: `sudo ifconfig <IFACE> down` where <IFACE> represents the interface name (eg. `enp6s0f0`)
2) Navigate to the `/local/onvm/openNetVM/scripts` directory and bind the NICs to DPDK using the command `source ./setup_cloudlab.sh`
3) Ensure that you see the two NICs in the section defining “Network devices using DPDK-compatible driver.” If you only see one NIC, it’s possible that you did not unbind the other NIC from the kernel driver using `sudo ifconfig <IFACE> down`. Repeat step (i).
4) Navigate back to the openNetVM folder (`cd ..`) and compile the Manager using `cd onvm && make && cd ..`
5) It is recommended to compile the NFs as well: `cd examples && make && cd ..`

Unrelated, but I've adjusted the link to the topology template so that it is now relative. This hopefully will allow the template link to reference the corresponding branch of the ONVM repo after merging.

Reply:
@jettjacobs Those shortened instructions are perfect. About the new ARP command, I apologize I should have been more specific. The command that is given in the topology template was wrong (the 2 ips were in the wrong order), so Jiyan made a new PDF to use. I will DM you the new PDF to use over Slack after I make this comment. I would check the PDF real quick before you add it to the repo to make sure you don't see any issues. 

Also, I forgot to include one other change to the documentation. Jiyan and I have been using a new load balancer profile that Jiyan created. It is called ONVM_LoadBalancer, and I will send you the link to the profile over Slack soon. That link should probably be updated in the Three Node Tutorial .rst file and references to the profile should be updated in both the Three Node Tutorial and Load Balancer Tutorial .rst files.